src/components/erc721/collectibles/collectibles_list.tsx

```diff
@@ -118,7 +122,9 @@ export class CollectiblesList extends React.Component<Props, State> {
                     <Title>{title}</Title>
                     <CollectiblesListSortStyled currentValue={sortType} onChange={this._onChangeSortType} />
                     <CollectiblesListFilterStyled currentValue={filterType} onChange={this._onChangeFilterType} />
-                    <ButtonStyled variant="quaternary">Sell collectibles</ButtonStyled>
+                    <ButtonStyled variant="quaternary" onClick={this.props.toggleCollectibleListModal}>
+                        Sell collectibles
+                    </ButtonStyled>
```

Comment:
We can probably make this button a component itself so its parent doesn't need to map the dispatch and store properties.

This might be useful for the landing page of the erc721 app. We can do it in this PR or after #428 is merged (I don't want to add a dependency here).